spec/unit/actions/synchronize_users_spec.rb

```diff
@@ -44,8 +48,25 @@
         GoodData::LCM2.convert_to_smart_hash(params)
       end
 
-      it_behaves_like 'a user action reading client_id' do
-        let(:client_id) { '123456789' }
+      context 'when the input set is empty' do
```

Comment:
Similar code found in synchronize_users_spec.rb:51, synchronize_users_spec.rb:61 (mass = 64)